sql/patch/00_STAT_STATEMENTS_DATA.sql

```diff
@@ -0,0 +1,8 @@
+SET statement_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+ALTER TABLE stat_statements_data
+   ADD COLUMN ssd_user_id integer DEFAULT 0;
```

Comment:
Do you mean firstly i add a nullable column then update the field and then make it not null?
If not please give me a simple code snippet about implementation.
Thanks.